cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/detector/SlowBrokerFinder.java

```diff
@@ -112,19 +113,26 @@ public SlowBrokerFinder() {
     // Preprocess raw metrics to get the derived metrics for each broker.
     Map<BrokerEntity, List<Double>> historicalValueByBroker = new HashMap<>();
     Map<BrokerEntity, Double> currentValueByBroker = new HashMap<>();
+    Set<Integer> skippedBrokers = new HashSet<>();
```

Comment:
(Not introduced in this patch) Nit: Can we make `SELF_HEALING_UNFIXABLE_RATIO` a `final`?